### ISSUE TITLE
loss function  hyperparameter

### ISSUE THREAD

**Issue by leowl0924** [albrateanu/LYT-Net#22]
in losses.py,   
        self.perceptual_loss_model = VGGPerceptualLoss(device)
        self.alpha1 = 1.00
        self.alpha2 = 0.06
        self.alpha3 = 0.05
        self.alpha4 = 0.5
        self.alpha5 = 0.0083
        self.alpha6 = 0.25
total_loss = (self.alpha1 * smooth_l1_l + self.alpha2 * perc_l + self.alpha3 * hist_l + self.alpha5 * psnr_l + self.alpha6 * color_l + self.alpha4 * ms_ssim_l)
there is no correspondence with the one in the paper, is there an error here?

**Comment by albrateanu** [comment]
Hi @leowl0924 
Thanks. Indeed the published code was not correct. I have amended the loss function weights in the latest commits.
